src/test/resources/application-test.yml

```diff
@@ -7,6 +7,9 @@ spring:
   flyway:
     enabled: false
 
+  cache:
```

Comment:
Could you add a comment here explaining we disable the cache because there are several tests relying on specific user access set-ups which would be ignored if there is data cached, please?